yo/app/scripts/model.js

```diff
@@ -0,0 +1,50 @@
+/*jshint undef: false, unused: false, indent: 2*/
```

Comment:
http://jshint.com/docs/
The ng-sortable plugin uses it, so I kept it (when removing it the netbeans editor complains about angular variable not defined).